libs/pages/services/src/lib/feature/page-new-feature/page-new-feature.tsx

```diff
@@ -100,7 +100,18 @@ function CardOption({ parentSlug, slug, icon, title, description, type, recommen
   )
 }
 
-function CardService({ title, icon, description, slug, options, type, link }: ServiceTemplateType) {
+function CardService({
+  title,
+  icon,
+  description,
+  slug,
+  options,
+  type,
+  link,
+  availableTemplates,
+}: ServiceTemplateType & {
+  availableTemplates: LifecycleTemplateListResponseResultsInner[]
```

Comment:
Maybe better to add directly on the `ServiceTemplateType` level with optional?

Reply:
I don't think this is a good idea. `ServiceTemplateType` represents one entry of the service templates list.
It doesn't make sense that each entry own an `availableTemplates` list.
This is why I chose to put it that way.